smriprep/cli/run.py

```diff
@@ -294,38 +278,23 @@ def build_workflow(opts, retval):
     from nipype import logging, config as ncfg
     from ..__about__ import __version__
     from ..workflows.base import init_smriprep_wf
+    from ..utils import Spaces
     from niworkflows.utils.bids import collect_participants
 
-    # Set the default template to 'MNI152NLin2009c'
-    output_spaces = opts.output_spaces or OrderedDict([('MNI152NLin2009cAsym', {})])
+    spaces = Spaces(output=opts.output_spaces)
```

Comment:
Will write a PR to your branch integrating the solution I have just proposed in NiWorkflows